docs/source-2.0/quickstart.rst

```diff
@@ -272,37 +287,27 @@ Let's define the operation used to "read" a ``City``.
 
     @readonly
     operation GetCity {
-        input: GetCityInput
-        output: GetCityOutput
-        errors: [NoSuchResource]
-    }
-
-    @input
-    structure GetCityInput for City {
-        // "cityId" provides the identifier for the resource and
-        // has to be marked as required.
-        @required
-        $cityId
-    }
-
-    @output
-    structure GetCityOutput {
-        // "required" is used on output to indicate if the service
-        // will always provide a value for the member.
-        @required
-        @notProperty
-        name: String
+        input := for City {
+            // "cityId" provides the identifier for the resource and
+            // has to be marked as required.
+            @required
+            $cityId
+        }
 
-        @required
-        $coordinates
-    }
+        output := for City {
+            // "required" is used on output to indicate if the service
+            // will always provide a value for the member.
+            @required
+            @notProperty
```

Comment:
We use `@notProperty`, but never say what it does like we do for `@required`.

Reply:
Added a comment on what the `notProperty` trait shows